src/components/select-project.html

```diff
@@ -1,9 +1,3 @@
-
-<link rel="import" href="shared-styles.html">
-<link rel="import" href="../bower_components/vaadin-combo-box/vaadin-combo-box.html">
```

Comment:
Even though `vaadin-combo-box` and `iron-ajax` are imported in a parent element, it's a good practice to always import dependencies inside the element that uses them. This is done to avoid breakage if there are changes at some point the parents' imports. Browser takes care of not importing anything multiple times so it's OK to have duplicates.